components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -32,6 +33,11 @@ using ai_chat::mojom::CharacterType;
 using ai_chat::mojom::ConversationTurn;
 using ai_chat::mojom::ConversationTurnVisibility;
 
+namespace {
+static const char* kAllowedSchemes[] = {url::kHttpsScheme, url::kHttpScheme,
```

Comment:
Is `base::MakeFixedFlatSet` better for a list of things we're only using for comparing via `base::Contains`?

Reply:
I can't seem to mark the container as `constexpr` as `url::..` are not allowed as constant expression which makes me think if it's really better than the previous approach. I don't see any advantage over a `static const char* []`

However, I still went ahead and changed it to the fixed container as you suggested. If you've notice I'm using string_view rather than StringPiece which is going to be an alias in the upcoming versions based on this thread: https://groups.google.com/a/chromium.org/g/chromium-dev/c/Yvqq8eaCshk

Reply:
When chromium is referencing schemes in a fixed flat set, they either do `static const` or put the string value directly in.

Reply:
added `static const`